builtin/logical/ssh/backend.go

```diff
@@ -20,6 +21,8 @@ type backend struct {
 	view      logical.Storage
 	salt      *salt.Salt
 	saltMutex sync.RWMutex
+	// Write lock around issuers
+	issuersLock sync.RWMutex
```

Comment:
```suggestion
	issuersLock sync.Mutex
```

I don't think we use the read lock anywhere?